app/stac_api/serializers.py

```diff
@@ -735,12 +735,21 @@ def update(self, instance, validated_data):
         return super().update(instance, validated_data)
 
     def validate(self, attrs):
-        validate_item_properties_datetimes(
-            attrs.get('properties_datetime', None),
-            attrs.get('properties_start_datetime', None),
-            attrs.get('properties_end_datetime', None),
-            partial=self.partial
-        )
+        if (
+            not self.partial or \
+            'properties_datetime' in attrs or \
+            'properties_start_datetime' in attrs or \
+            'properties_end_datetime' in attrs
+        ):
+            validate_item_properties_datetimes(
+                attrs.get('properties_datetime', None),
```

Comment:
So `None`is a valid date?

Reply:
None is not a valide date, but in properties there is 3 dates:
- properties_datetime
- properties_start_datetime
- properties_end_datetime

and either the first one is required or both start and end datetime, that's what validate the function.